--- origsrc/cream-0.41/INSTALL.sh	2006-11-02 00:20:30.000000000 -0600
+++ src/cream-0.41/INSTALL.sh	2008-04-24 15:43:09.937500000 -0500
@@ -12,9 +12,9 @@
 fi
 
 # find $VIMRUNTIME if doesn't exist (no trailing slash)
-if [ "$VIMRUNTIME" = "" ] || [ -d $VIMRUNTIME ] ; then
+if [ "$VIMRUNTIME" = "" ] || [ ! -d $VIMRUNTIME ] ; then
     # warn only if not directory
-	if [ -d $VIMRUNTIME ] ; then
+	if [ ! -d $VIMRUNTIME ] ; then
 		echo "  Existing \$VIMRUNTIME not a directory."
 	fi
 	echo "Finding \$VIMRUNTIME..."
@@ -297,10 +297,6 @@
 if [ -e "$VRT/cream/docs-html/otherfiles.html" ]; then
 	rm -f $VRT/cream/docs-html/otherfiles.html
 fi
-# 0.33
-if [ -e "/usr/bin/cream" ]; then
-	rm -f /usr/bin/cream
-fi
 # 0.31
 if [ -e "$VRT/cream/docs-html/love.html" ]; then
 	rm -f $VRT/cream/docs-html/love.html
